### PR TITLE
dirac: add metadata method in add_files for dirac #6437

### PR DESCRIPTION
**Pull request by panta-123**
Lets get some comments/green light on the idea. 
Especially from those using rucio and dirac.

@cserf , Is this acceptable from Belle II point of view ?

### CODE REVIEW

**Comment by panta-123** [comment]
The add_files method with metadata , test passed.
`[gw1] [ 57%] PASSED tests/test_belleii.py::test_dirac_addfile_with_parents_meta `.

Changing PR to "ready for review"

**Comment by cserf** [comment]
A few tests are failing (syntax checks and docs), but except that, looks good to to me. Can be merged when the test are fixed

**Comment by panta-123** [comment]
All the check passed. 
The failed End to END integration test is not related to this PR.

**Review by cserf — DISMISSED** [review]

File: lib/rucio/api/dirac.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 
 @transactional_session
-def add_files(lfns, issuer, ignore_availability, vo='def', *, session: "Session"):
```
Comment (bari12) [inline lib/rucio/api/dirac.py:31]: Add `parents_metadata` to the method description (param) below

File: lib/rucio/client/diracclient.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ class DiracClient(BaseClient):
 
     DIRAC_BASEURL = 'dirac'
 
-    def add_files(self, lfns, ignore_availability=False):
```
Comment (bari12) [inline lib/rucio/client/diracclient.py:31]: Add parameter to description

**Review by bari12 — CHANGES_REQUESTED** [review]

File: lib/rucio/api/dirac.py (hunk 1 above)
Comment (panta-123) [reply lib/rucio/api/dirac.py:31]: done

File: lib/rucio/client/diracclient.py (hunk 2 above)
Comment (panta-123) [reply lib/rucio/client/diracclient.py:31]: done

**Review by bari12 — APPROVED** [review]
I am merging this as an enhancement within the `33` release line.

**Comment by bari12** [comment]
Also please squash commits next time :-)

**Pull request merged**